server/lib/packets/packet.go

```diff
@@ -4,9 +4,10 @@ import (
 	"log"
 	"fmt"
 
-	"../assets"
-	"../models"
-	proto "../../proto"
+	"github.com/umegaya/yue"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/yue (can't find import: github.com/umegaya/yue)